lib_eio/unix/eio_unix.mli

```diff
@@ -84,6 +84,20 @@ module Stdenv : sig
       You can use the functions in {!Eio.Stdenv} to access these fields if you prefer. *)
 end
 
+module Sockopt : sig
```

Comment:
Would probably be better as `Eio_unix.Net.Sockopt`.

Reply:
Done in 92a06fe38597fae48c2f520d2d7d070e62cf23a3